### DISCUSSION TITLE
Logs data was not removed after deleted the last query field in the Loki query dashboard

### DISCUSSION THREAD

**Discussion by SanthoshS20** [grafana/grafana#53713] [Observability]
**Problem/Idea:**
In the Loki query dashboard, Logs data was not removed after deleted/removed the last query field in the dashboard. Queried logs was still present in the dashboard. If we run the Run Query button, this results in showing empty log volume after the last data present. Sometimes If a user wants to start from the scratch, they will reload the explore page.

**Impact:**
This impacts the customers. The customers will reload the Explore page again to start from the scratch. This results in accessing API request.

**Solution:**
Once the last query field is removed, it should remove all the log volume and logs data from the dashboard. This helps the user to start from the scratch. 

**Steps:**
1. Go to the Loki query dashboard.
2. Add a query in the query field and run it. 
3. Verify the logs present for the query.
4. Delete the query field. 
5. logs and log volume will be present.
6. Try execute the Run Query button. This results in adding empty data to the log volume.

**Image:**
![image](https://user-images.githubusercontent.com/37366893/184528701-2456af8b-a418-4eed-89e2-24ff712717ca.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
